core/fee/engine.go

```diff
@@ -424,20 +425,16 @@ func (e *Engine) BuildLiquidityFeeDistributionTransfer(shares map[string]num.Dec
 		ft.transfers = append(ft.transfers, &types.Transfer{
 			Owner: key,
 			Amount: &types.FinancialAmount{
-				Amount: amount.Clone(),
+				Amount: amount,
 				Asset:  acc.Asset,
 			},
 			MinAmount: amount.Clone(),
 			Type:      types.TransferTypeLiquidityFeeDistribute,
 		})
 	}
 
+	// if there is a remainder, just keep it in the fee account, will be used next time we pay out fees
 	// last is the party who will get the remaining from ceil
```

Comment:
this is no longer true I reckon.

Reply:
Yes it is, that's the whole point of the change